tidb-cloud/explore-data-with-chat2query.md

```diff
@@ -0,0 +1,110 @@
+
+    - If you have multiple queries in the editor, to run one or several of them sequentially, select the lines of the target queries with your cursor, and then press **Command + Enter** or click **Run**.
+
+    - To run all queries in the editor sequentially, press **Shift + Command + Enter**, or select the lines of all queries with your cursor and click **Run**.
+
+    </div>
+
+    <div label="Windows/Linux">
+
+    For Windows or Linux:
+
+    - If you have only one query in the editor, to run it, press **Ctrl + Enter** or click <svg width="1rem" height="1rem" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M6.70001 20.7756C6.01949 20.3926 6.00029 19.5259 6.00034 19.0422L6.00034 12.1205L6 5.33028C6 4.75247 6.00052 3.92317 6.38613 3.44138C6.83044 2.88625 7.62614 2.98501 7.95335 3.05489C8.05144 3.07584 8.14194 3.12086 8.22438 3.17798L19.2865 10.8426C19.2955 10.8489 19.304 10.8549 19.3126 10.8617C19.4069 10.9362 20 11.4314 20 12.1205C20 12.7913 19.438 13.2784 19.3212 13.3725C19.307 13.3839 19.2983 13.3902 19.2831 13.4002C18.8096 13.7133 8.57995 20.4771 8.10002 20.7756C7.60871 21.0812 7.22013 21.0683 6.70001 20.7756Z" fill="currentColor"></path></svg>**Run**.
+
+    - If you have multiple queries in the editor, to run one or several of them sequentially, select the lines of the target queries with your cursor, and then press **Ctrl + Enter** or click **Run**.
+
+    - To run all queries in the editor sequentially, press **Shift + Ctrl + Enter**, or select the lines of all queries with your cursor and click **Run**.
+
+    </div>
+    </SimpleTab>
+
+After running the queries, you can see the query log and results immediately at the bottom of the page.
+
+## Manage SQL files
+
+In Chat2Query, you can save your SQL queries in different SQL files and manage SQL files as follows:
+
+- To add a SQL file, click **+** on the **SQL Files** tab.
+- To rename a SQL file, move your cursor on the filename, click **...** next to the filename, and then select **Rename**.
+- To delete a SQL file, move your cursor on the filename, click **...** next to the filename, and then select **Delete**.
+
+## Manage Chat2Query settings
+
+By default, Chat2Query limits the maximum number of rows in query results to 500 and does not show system database schemas on the **Schemas** tab.
+
+To change the settings, take the following steps:
+
+1. In the upper-right corner of Chat2Query, click **...** and select **Chat2Query settings**.
```

Comment:
```suggestion
1. In the upper-right corner of Chat2Query, click **...** and select **Settings**.
```
<img width="193" alt="image" src="https://user-images.githubusercontent.com/60599231/211495587-63d763f1-f9ed-4845-941d-cdb92a64ce85.png">

Reply:
Good catch. It was newly updated today.